app/models/local_authority.rb

```diff
@@ -23,11 +23,11 @@ def signatory
   end
 
   def council_name
-    plan_x? ? subdomain.capitalize : "#{subdomain.capitalize} Council"
```

Comment:
For PlanX I switched this to use `council_code` rather than `capitalize` or `titleize` since `Planx` would've been the result.